DashboardClient/DashboardClient.cpp

```diff
@@ -383,14 +422,18 @@ namespace Umati
 			std::map<std::shared_ptr<const ModelOpcUa::Node>, nlohmann::json> &valueMap,
 			std::mutex &valueMap_mutex)
 		{
-			// LOG(INFO) << "subscribeValues "   << pNode->NodeId.Uri << ";" << pNode->NodeId.Id;
+			LOG(INFO) << "subscribeValues "   << pNode->NodeId.Uri << ";" << pNode->NodeId.Id;
+			for(auto value : m_subscribedValues){
```

Comment:
Can you make a comment, why the loop is nessasary? I think you check if the values is already subscribed? Is this correct?

Reply:
Yes.